metabrainz/config.py.example

```diff
@@ -48,6 +48,14 @@ MUSICBRAINZ_CLIENT_ID = ""
 MUSICBRAINZ_CLIENT_SECRET = ""
 
 
+# ADMIN SECTION
+# This is a list of MusicBrainz usernames of people that are allowed to access
+# admin section of the website. Usernames are case-sensetive!
+ADMINS = [
+    'rob',
+]
```

Comment:
Please add yourself as well. It will be nice to have two people be able to do things.